_includes/translated_links.html

```diff
@@ -1,20 +1,18 @@
 {% if page.i18n_key %}
-  {% if page.layout == 'single_event' %}
+  {% if page.layout == 'single_post' %}
     {% assign posts=site.posts | where:"i18n_key", page.i18n_key %}
     {% assign page_language = page.categories.first | default: page.lang %}
   {% else %}
     {% assign posts=site.pages | where:"i18n_key", page.i18n_key %}
-    {% assign page_language = page.dir | replace: '/' %}
+    {% assign page_language = page.dir | replace: '/' | slice: 0, 2 | default: "en" %}
   {% endif %}
-
   {% if posts.size > 1 %}
     {% for post in posts %}
-      {% if page.layout == 'single_event' %}
+      {% if page.layout == 'single_post' %}
         {% assign post_language=  post.categories.first %}
       {% else %}
-        {% assign post_language=  {{post.dir}} | replace: '/' %}
+        {% assign post_language=  post.dir | replace: '/' | slice: 0,2 | default: "en" %}
```

Comment:
for documentation purposes, this lead to situations where on the home page of `/`, the language code was `/` while the posts on it had language code of `/en`, so we give it a default of `en` when it's empty.. With a more rich url like `/en/events/` it will strip the `/` and select the first two letters, which is `en` in this case, and perhaps `/de/termine` and still `de` as the german prefix